### PR TITLE
fix: fixed typo, added some minor changes in naming

### PR DESCRIPTION
**Pull request by alexander-mobilab**

### CODE REVIEW

File: src/redux/sagas/saga.js [hunk 1]
```diff
@@ -32,10 +32,10 @@ function* watchAddAlbum(){
 
 
 
-function* fetchImages(ID) {
+function* fetchImages(albumID) {
```
Comment (miladnavi) [inline src/redux/sagas/saga.js]: As usual var names should be camel case e.g. albumId

File: src/redux/sagas/saga.js [hunk 2]
```diff
@@ -32,10 +32,10 @@ function* watchAddAlbum(){
...
     let albumImages = [];
       try {
-        const response = yield fetch('https://api.imgur.com/3/album/'+ID, {
+        const response = yield fetch('https://api.imgur.com/3/album/'+albumID, {
         method: 'GET',  
         headers: {
           'Authorization': 'Client-ID 77248e7a549713b'},
```
Comment (miladnavi) [inline src/redux/sagas/saga.js]: Try to use the Javascript function to iterate an array like (forEach, filter, ...)

File: src/redux/sagas/saga.js [hunk 3]
```diff
@@ -51,7 +51,7 @@ function* fetchImages(ID) {
 }
 
 function* addImagesAsync(action) {
-    const img = yield fetchImages(action.ID);
+    const img = yield fetchImages(action.albumID);
     yield put({type: 'ADD_IMAGE_ASYNC', img});
 }
```
Comment (miladnavi) [inline src/redux/sagas/saga.js]: 1- Try to split your code into different files (AddAlbum, AddImage)
2- Functions like rootSaga belongs to index files

File: src/redux/store/reducer.js [hunk 4]
```diff
@@ -2,7 +2,7 @@ import { combineReducers } from 'redux';
 
 
 const initialState = {
-  user: 'StormWingDelta',
+  user: 'alexmobi',
   albumList: [],
   imageList: []
 }
```
Comment (miladnavi) [inline src/redux/store/reducer.js]: could you explain   
> const newState = {...state};

* reducer should be implemented with switch cases and each case should return proper state
*

File: src/redux/store/reducer.js [hunk 5]
```diff
@@ -13,7 +13,7 @@ const myReducer = (state = initialState, action) => {
   if(action.type === 'ADD_ALBUMS_ASYNC'){
     newState.albumList = action.albums;
   }
-  if(action.tyoe === 'CLEAR_ALBUMS'){
+  if(action.type === 'CLEAR_ALBUMS'){
     newState.albumList = [];
   }
```
Comment (miladnavi) [inline src/redux/store/reducer.js]: * For should be replaced with javascript functions like (forEach, filter, ...)
* You can manage to set state in ADD_IMAGE_ASYNC in sags, you don't need to iterate the Array again. (Each iteration cost you time)
* Don't forget to do ...state for property in the state which should stay unchanged for each action

File: src/screens/Albums/HomeScreen.js [hunk 6]
```diff
@@ -24,7 +24,7 @@ class HomeScreen extends Component {
             <Button title={item.title}
             onPress={() => {
               this.props.clearImages();
-              this.props.navigation.navigate('Images', { itemID: item.ID });
+              this.props.navigation.navigate('Images', { albumID: item.ID });
             }}
             />}
             keyExtractor={(item, index) => index.toString()}
```
Comment (miladnavi) [inline src/screens/Albums/HomeScreen.js]: * You can set the image Id as key

File: src/screens/Albums/HomeScreen.js [hunk 7]
```diff
@@ -24,7 +24,7 @@ class HomeScreen extends Component {
             <Button title={item.title}
```
Comment (miladnavi) [inline src/screens/Albums/HomeScreen.js]: * Inline Style is not clean and style should be in a separate file

File: src/redux/sagas/saga.js (hunk 1 above)
Comment (alexander-mobilab) [reply src/redux/sagas/saga.js]: Changed it everywhere.

File: src/screens/Albums/HomeScreen.js (hunk 6 above)
Comment (alexander-mobilab) [reply src/screens/Albums/HomeScreen.js]: Changed it to: keyExtractor={(item) => item.id}

File: src/screens/Albums/HomeScreen.js (hunk 7 above)
Comment (alexander-mobilab) [reply src/screens/Albums/HomeScreen.js]: Done.

File: src/redux/sagas/saga.js (hunk 3 above)
Comment (alexander-mobilab) [reply src/redux/sagas/saga.js]: Done.

File: src/redux/store/reducer.js (hunk 4 above)
Comment (alexander-mobilab) [reply src/redux/store/reducer.js]: Done.

File: src/redux/sagas/saga.js (hunk 2 above)
Comment (alexander-mobilab) [reply src/redux/sagas/saga.js]: Error will return empty array now. An empty array will trigger a default message on screen.

**Pull request merged**